frontend/packages/kubevirt-plugin/src/components/vmtemplates/vm-template.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import * as classNames from 'classnames';
+import { sortable } from '@patternfly/react-table';
+
+import {
+  TemplateSource,
+  getTemplateOperatingSystems,
+  getTemplateFlavors,
+} from 'kubevirt-web-ui-components';
+
+import { ListPage, Table, TableRow, TableData } from '@console/internal/components/factory';
+import { Kebab, ResourceLink, ResourceKebab } from '@console/internal/components/utils';
+import { VmTemplateModel } from '../../models';
+
+export const menuActions = Kebab.factory.common;
+
+const { kind, selector } = VmTemplateModel;
+
+const tableColumnClasses = [
+  classNames('col-lg-2', 'col-sm-4', 'col-xs-4'),
+  classNames('col-lg-2', 'hidden-sm', 'hidden-xs'),
+  Kebab.columnClass,
+];
+
+const VmTempleateTableHeader = () => {
+  return [
+    {
+      title: 'Name',
+      sortField: 'metadata.name',
+      transforms: [sortable],
+      props: { className: tableColumnClasses[0] },
+    },
+    {
+      title: 'Namespace',
+      sortField: 'metadata.namespace',
+      transforms: [sortable],
+      props: { className: tableColumnClasses[0] },
```

Comment:
IMO it would be better to map the same index of tableColumnClasses to the Header index

- it is easier to read and make changes to

Reply:
Ahhh ... wow, I saw it in the example ... and didn't understand what i saw :blush: 

Thanks, will fix that  :exclamation: